src/components/AllCitiesOverviewContainer/AllCitiesOverview/AllCitiesOverview.scss

```diff
@@ -123,36 +123,57 @@ $round-image-width: 220px;
   font-weight: 500;
 }
 
+.subInfoWrapper {
+  @include shadow-inset-bottom;
+
+  background-color: $color-primary--020;
+}
+
 .subInfoContainer {
   @include container;
 
-  display: flex;
-  flex-direction: column;
   padding-top: 42px;
   padding-bottom: 42px;
+}
 
-  @include for-tablet-portrait-up {
-    flex-direction: row;
-  }
+.legend {
+  margin: 0 0 12px;
+  font-size: 22px;
+  font-weight: 300;
 }
 
-.subInfoTextWrapper {
-  flex: auto;
+.legendRow {
   margin-bottom: 32px;
 
-  @include for-tablet-portrait-up {
+  &:last-child {
     margin-bottom: 0;
-    margin-right: 64px;
+  }
+
+  @include for-tablet-landscape-up {
+    display: flex;
+    margin-bottom: 10px;
+  }
+}
+
+.icon {
+  margin-right: 14px;
```

Comment:
This icon doesn't quite align correctly. Its vertical-align is middle but the other text will default to baseline. You can set
```css
vertical-align: -8px;
```

To align it nicely.

Reply:
Can also set to 100% high to avoid a hardcoded pixel value.

Reply:
I did actually notice that alignment thing previously but forgot to fix it, whoops!